pkg/arvo/sys/vane/gall.hoon

```diff
@@ -421,8 +421,20 @@
       (drop (bind (~(get by yokes.state) u.dude) (lead u.dude)))
     |-  ^+  mo-core
     ?~  apps  mo-core
-    =/  ap-core  (ap-yoke:ap:mo-core p.i.apps [~ our] q.i.apps)
```

Comment:
I probably didn't need to change this line, but it seems less brittle without actually firing the `+mo-core` arm.